contracts/interface/HolographTreasuryInterface.sol

```diff
@@ -101,6 +101,4 @@
 
 pragma solidity 0.8.13;
 
-interface HolographTreasuryInterface {
-
-}
+interface HolographTreasuryInterface {}
```

Comment:
@alexanderattar @BravoNatalie @ACC01ADE Is there anyway we can make this not happen anymore?

Reply:
I believe it's getting modified via the transpilation step and then prettier converts it back. I stopped running prettier:fix because it was messing with foundry's git submodule dependencies so i'm not sure how it happened here. probably need to format it properly via prettier during the transpilation step, but would need to look into it more